examples/with-nextjs/pages/api/search.ts

```diff
@@ -1,5 +1,24 @@
-import { searchHandler } from 'postgres-searchbox';
+import { getSearchHandler } from 'postgres-searchbox';
 // During postgres-searchbox development this can be:
 // import { searchHandler } from '../../../../package/build';
 
-export default searchHandler;
+export default getSearchHandler([
```

Comment:
This was my first attempt at passing a server-side config. I've sice changed it to match Algolia's api. With the benefits that their docs and TS types are aligned with this package. The revised config is in https://github.com/dekimir/postgres-searchbox/pull/12